site/src/pages/author.js

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import Author from "../sections/author";
+
+const IndexPage = () => {
+  return (
+    <>
+      <Author />
+    </>
+  );
```

Comment:
Add the `SeoComponent` and the `Layout` component. Check the `index.js` file for reference,